.swiftlint.yml

```diff
@@ -42,6 +42,9 @@ variable_name:
     - rs
     - URI
     - URL
+type_name:
+  excluded:
+    - T
```

Comment:
Though, maybe really we should just name that type `TestableItem` instead of `T`.